packages/blade/docs/guides/Installation.stories.mdx

```diff
@@ -47,26 +47,22 @@ Blade has a peer dependency on a few libraries, you can skip adding it if you al
   > **Note**
   >
   > Currently, blade only supports styled-components v5 only
-  - `@floating-ui/react`
+
    ```shell
-   yarn add @razorpay/blade styled-components@5.3.11 @floating-ui/react
+   yarn add @razorpay/blade styled-components@5.3.11
    ```
 2. Follow [this guide](#-install-fonts) to install the fonts.
 
 3. For **React Native** projects you need to do additional setup for the peer dependencies: 
 
     - react-native-reanimated
-      - Follow [this guide](https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/installation) to install it on Android & iOS which is required by Blade.
-    - react-native-svg
-      - Follow [this guide](https://github.com/react-native-svg/react-native-svg#with-react-native-cli) to install it on Android & iOS which is required by Blade.
-    - react-native-gesture-handler
-      - Follow [this guide](https://docs.swmansion.com/react-native-gesture-handler/docs/installation) to install it, note that you don't need to add `<GestureHandlerRootView style={{ flex: 1 }}>` again on the root because BladeProvider already adds that out of the box.
-    - @gorhom/bottom-sheet
```

Comment:
for RN we shall keep it as it is? because for web these dependencies will get installed so website team might raise concern about increased installation time and sometimes these libs might even fail since they won't have RN specific things in env installed. so probably we shall not have RN dependencies in our dependencies by default?

Reply:
Moved RN stuff to peer.